plugins/genesys-gl32xx/fu-genesys-gl32xx-device.c

```diff
@@ -34,15 +34,18 @@ static gboolean
 fu_genesys_gl32xx_device_send_scsi_cmd_v3(FuGenesysGl32xxDevice *self,
 					  const guint8 *cdb,
 					  guint8 cdbsz,
-					  const guint8 *buf,
-					  guint32 bufsz,
+					  const guint8 *inbuf,
+					  gsize inbufsz,
+					  guint8 *outbuf,
+					  gsize outbufsz,
```

Comment:
I think the usage of `buf` is case by `FuGenesysGl32xxDirection`. With `FU_GENESYS_GL32XX_DIRECTION_OUT`, host sends `buf` to device, and `FU_GENESYS_GL32XX_DIRECTION_IN`, host received `buf` from device. Therefore,

Reply:
Yes, good idea -- fixed.